src/main.c

```diff
@@ -585,7 +586,7 @@ static gboolean extract_start(int argc, char **argv)
 	g_debug("input bundle: %s", argv[2]);
 	g_debug("output dir: %s", argv[3]);
 
-	if (!check_bundle(argv[2], &bundle, TRUE, &ierror)) {
+	if (!check_bundle(argv[2], &bundle, !disable_security_checks, &ierror)) {
```

Comment:
This is what `--no-verify` does in other commands: disable signature verification. Why do you use a different name for extract? Or is the goal to only disable the concurrent access checks (as @ejoerns does in https://github.com/rauc/rauc/pull/770)?

Reply:
> This is what --no-verify does in other commands: disable signature verification.

That's true. But it has further implications than skipping the signature verification. Hence a more apt name.

> Or is the goal to only disable the concurrent access checks (as @ejoerns does in #770)?

Exactly. The primary intention is to disable the concurrent access checks. I'd say that #770 seems to be the way to go. And once it's merged, it should be quite straight forward to be supported by the `extract` command to.